src/extensions/default/bramble/lib/RemoteCommandHandler.js

```diff
@@ -194,6 +194,30 @@ define(function (require, exports, module) {
             skipCallback = true;
             CommandManager.execute("bramble.addCodeSnippet", args[0]).always(callback);
             break;
+        case "CODEMIRROR_RELAY":
+            var editor = EditorManager.getActiveEditor();
```

Comment:
We need to move all this into the editor, probably `Editor.js`, vs. doing it across the bramble `postMessage` API.